src/nvim/tui/tui.c

```diff
@@ -16,6 +16,11 @@
 #include "nvim/os/event.h"
 #include "nvim/tui/tui.h"
 
+// Space reserved in the output buffer to restore the cursor to normal when
```

Comment:
> It just seems strange saying "Bytes reserved in the output buffer" since whats reserved are the slots that will contain the bytes

But you're allocating `OUTBUF_SIZE` bytes, right? Or is that not what you meant by `Space reserved`?

> By saying "No existing terminal will require 32 bytes to do that" I meant that there are no cnorm codes that take 32 bytes of space.

Ah, my mistake.